tests/openai/test_openai_protocol.py

```diff
@@ -0,0 +1,189 @@
+import openai
+import pytest
+import os
+import subprocess
+from openai import OpenAI
+
+os.environ["no_proxy"] = "localhost,127.0.0.1"
+os.environ["OPENAI_API_BASE"] = "http://localhost:8000/v1"
+os.environ["OPENAI_API_KEY"] = "YOUR_OPEN_AI_KEY"
+os.environ["OPENAI_BASE_URL"] = "http://localhost:8000/v1"
+openai_base_url = os.environ["OPENAI_BASE_URL"]
+openai_api_key = os.environ["OPENAI_API_KEY"]
+client = OpenAI(base_url=openai_base_url, api_key=openai_api_key)
+
+
+def start_serve(model_name):
```

Comment:
The same function also appears in `test_example_simple.py`. Can we put it in a common place?

Reply:
ok,i will reorganize the function locations